### ISSUE TITLE
Can't create new env0_project + env0_template + env0_template_project_assignment (Terraform 1.0.0 Env0 Provider 0.0.9)

### ISSUE THREAD

**Issue by smsilva** [env0/terraform-provider-env0#120]
Terraform Version:

```
Terraform v1.0.0
on linux_amd64
+ provider registry.terraform.io/env0/env0 v0.0.9
```
Ubuntu version:

```
20.04.2
```

Hi, I'm start trying Env0 Terraform Provider and I'm getting an error when I try to run a Terraform Apply.

Here are the steps that I did:

1. Create an Env0 API Key.

2. Successfully tested the Credentials using `curl`:

```
curl \
  --silent \
  --header 'Accept: application/json' \
  --user ${ENV0_API_KEY?}:${ENV0_API_SECRET?} \
  "https://api.env0.com/blueprints?organizationId=${ENV0_ORGANIZATION?}" | jq
```
3. Set Environment Variables for test:

```
ENV0_ORGANIZATION=de6d1bb0-f746-4974-ab74-1ff73d0e2445
ENV0_API_SECRET=api_secret_value_here
ENV0_API_KEY=api_key_value_here
```

4. Create a Simple Env0 Project worked:

```
# main.tf
terraform {
  required_providers {
    env0 = {
      source = "env0/env0"
      version = "0.0.9"
    }
  }
}

provider "env0" {
}

resource "env0_project" "example" {
  name        = "example"
  description = "Example project"
}
```

5. When I'm trying to create a new env0_template and an env0_template_project_assignment using this:

```
terraform {
  required_providers {
    env0 = {
      source = "env0/env0"
      version = "0.0.9"
    }
  }
}

provider "env0" {
}

resource "env0_project" "example" {
  name        = "example"
  description = "Example project"
}

resource "env0_template" "example" {
  name        = "mytemplate"
  description = "Example template"
  repository  = "https://github.com/env0/templates"
  path        = "aws/hello-world"
}

resource "env0_template_project_assignment" "assignment" {
  template_id = env0_template.example.id
  project_id  = env0_project.example.id
}
```

Plan:

```
Terraform used the selected providers to generate the following execution plan. Resource actions are indicated with the following symbols:
  + create

Terraform will perform the following actions:

  # env0_project.example will be created
  + resource "env0_project" "example" {
      + description = "Example project"
      + id          = (known after apply)
      + name        = "example"
    }

  # env0_template.example will be created
  + resource "env0_template" "example" {
      + description       = "Example template"
      + id                = (known after apply)
      + name              = "mytemplate"
      + path              = "aws/hello-world"
      + repository        = "https://github.com/env0/templates"
      + terraform_version = "0.15.1"
      + type              = "terraform"
    }

  # env0_template_project_assignment.assignment will be created
  + resource "env0_template_project_assignment" "assignment" {
      + id          = (known after apply)
      + project_id  = (known after apply)
      + template_id = (known after apply)
    }

Plan: 3 to add, 0 to change, 0 to destroy.

────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────

Saved the plan to: .terraform.plan

To perform exactly these actions, run the following command to apply:
    terraform apply ".terraform.plan"
```

Error:

```
env0_project.example: Creating...
env0_template.example: Creating...
env0_project.example: Creation complete after 6s [id=a56c4001-e98d-4c66-9487-8724c4465073]
╷
│ Error: could not create template: 400 Bad Request: ["/body/retry/onDestroy must have required property times","/body/retry/onDestroy must be null","/body/retry/onDestroy must match a schema in \"anyOf\"","/body/retry/onDestroy must be null","/body/retry/onDestroy must match a schema in \"anyOf\"","/body/retry/onDeploy must have required property times","/body/retry/onDeploy must be null","/body/retry/onDeploy must match a schema in \"anyOf\"","/body/retry/onDeploy must be null","/body/retry/onDeploy must match a schema in \"anyOf\"","/body/retry must be null","/body/retry must match a schema in \"anyOf\""]
│ 
│   with env0_template.example,
│   on main.tf line 18, in resource "env0_template" "example":
│   18: resource "env0_template" "example" {
│ 
╵
```

Am I missing something?

Thanks.

**Comment by avnerenv0** [comment]
Hi @smsilva , looks like this is an issue on our side, taking a look ...

**Comment by avnerenv0** [comment]
@smsilva This was fixed in #117 , we just missed releasing that.
Was released in v0.0.10 available now in the registry.

Please let us know if using the newer version fixes your issue.

**Comment by smsilva** [comment]
It works! Thanks a lot!
